src/app/page.tsx

```diff
@@ -1,7 +1,11 @@
 'use client'
+import Donut from './components/DonutChart'
+import MiniDrawer from './components/sideBar'
+import NavBar from './components/NavBar'
+import Dashboard from './dashboard/page';
 const App = () => {
   return (
-      <></>
+      <><Dashboard/></>
```

Comment:
This page is only reserved for the landign page

Reply:
I did that for testing purposes. It's empty now.